tests/Aspire.Hosting.Tests/RabbitMQ/AddRabbitMQTests.cs

```diff
@@ -11,6 +11,27 @@ namespace Aspire.Hosting.Tests.RabbitMQ;
 
 public class AddRabbitMQTests
 {
+    [Fact]
+    public void AddRabbitMQAddsGeneratedPasswordParameterWithUserSecretsParameterDefaultInRunMode()
+    {
+        using var appBuilder = TestDistributedApplicationBuilder.Create();
+
+        var rmq = appBuilder.AddRabbitMQ("rmq");
+
+        Assert.IsType<UserSecretsParameterDefault>(rmq.Resource.PasswordParameter.Default);
+    }
+
+    [Fact]
+    public void AddRabbitMQDoesNotAddGeneratedPasswordParameterWithUserSecretsParameterDefaultInPublishMode()
+    {
+        using var appBuilder = TestDistributedApplicationBuilder.Create(DistributedApplicationOperation.Publish);
+        appBuilder.Environment.EnvironmentName = "Test";
```

Comment:
(nit) The test is named `InPublishMode()` but the environment name is `Test`. Can we either change the test name to be "NonRunMode()" or change the environment name to "Publish"?